src/ui/views/menu.coffee

```diff
@@ -1,74 +1,168 @@
-        { type: 'separator' }
-        { label: 'Hide YakYak', accelerator: 'Command+H', selector: 'hide:' }
-        { label: 'Hide Others', accelerator: 'Command+Shift+H', selector: 'hideOtherApplications:' }
-        { label: 'Show All', selector: 'unhideAllApplications:' }
-        { type: 'separator' }
-        { label: 'Open Inspector', accelerator: 'Command+Alt+I', click: -> action 'devtools' }
+acceleratorMap = {
+    hideyakyak:           {others: 'Control+H', darwin:'Command+H'}
+    hideothers:           {others: '', darwin:'Command+Shift+H'}
+    showall:              {others: '', darwin:''}
+    openinspector:        {others: 'Control+Alt+I', darwin:'Command+Alt+I'}
+    quit:                 {others: 'Control+Q', darwin:'Command+Q'}
+    undo:                 {others: 'Control+Z', darwin:'Command+Z'}
+    redo:                 {others: 'Control+Shift+Z', darwin:'Command+Shift+Z'}
+    cut:                  {others: '', darwin:'Command+X'}
+    copy:                 {others: '', darwin:'Command+C'}
+    paste:                {others: '', darwin:'Command+V'}
+    selectall:            {others: '', darwin:'Command+A'}
+    togglefullscreen:     {others: 'Control+Alt+F', darwin:'Command+Control+F'}
+    zoomin:               {others: 'Control+Plus', darwin:'Command+Plus'}
+    zoomout:              {others: 'Control+-', darwin:'Command+-'}
+    resetzoom:            {others: 'Control+0', darwin:'Command+0'}
+    previousconversation: {others: 'Control+K', darwin:'Command+Shift+Tab'}
+    nextconversation:     {others: 'Control+J', darwin:'Command+Tab'}
+    conversation1:        {others: 'Alt+1', darwin:'Command+1'}
+    conversation2:        {others: 'Alt+2', darwin:'Command+2'}
+    conversation3:        {others: 'Alt+3', darwin:'Command+3'}
+    conversation4:        {others: 'Alt+4', darwin:'Command+4'}
+    conversation5:        {others: 'Alt+5', darwin:'Command+5'}
+    conversation6:        {others: 'Alt+6', darwin:'Command+6'}
+    conversation7:        {others: 'Alt+7', darwin:'Command+7'}
+    conversation8:        {others: 'Alt+8', darwin:'Command+8'}
+    conversation9:        {others: 'Alt+9', darwin:'Command+9'}
+    minimize:             {others: '', darwin:'Command+M'}
+    close:                {others: '', darwin:'Command+W'}
+}
+
+templateYakYak = (viewstate, platform) ->
+    tmpl = []
+    if platform == 'darwin'
+        tmpl.push.apply tmpl,  [
+            { label: 'About YakYak', selector: 'orderFrontStandardAboutPanel:' }
+            #{ type: 'separator' }
+            # { label: 'Preferences...', accelerator: 'Command+,',
+            # click: => delegate.openConfig() }
+            { type: 'separator' }
+        ]
+    tmpl.push {
+        label: 'Hide YakYak'
+        accelerator: acceleratorMap['hideyakyak'][platform]
+        selector: 'hide:' if platform == 'darwin'
+        role: 'minimize' if platform == 'others'
+    }
+
+    if platform == 'darwin'
+        tmpl.push.apply tmpl, [{
```

Comment:
Anyway using array.concat() is more `functional` style as it avoids mutations: `tmpl = tmpl.concat([..])`
Also know that you can do `array.push(array...)` in coffeescript.
Since I don't think efficiency is relevant here maybe you could also push each item one by one.
I would avoid using `apply`.